examples/combined_ODNP.py

```diff
@@ -156,20 +156,21 @@
 ini_time = time.time()
 vd_data = None
 for vd_idx, vd in enumerate(vd_list_us):
+    # call A to run_IR
     vd_data = run_IR(
         nPoints=nPoints,
         nEchoes=config_dict["nEchoes"],
         indirect_idx=vd_idx,
         indirect_len=len(vd_list_us),
+        ph1_cyc=IR_ph1_cyc,
+        ph2_cyc=IR_ph2_cyc,
         vd=vd,
         nScans=config_dict["thermal_nScans"],
         adcOffset=config_dict["adc_offset"],
         carrierFreq_MHz=config_dict["carrierFreq_MHz"],
         p90_us=config_dict["p90_us"],
         tau_us=config_dict["tau_us"],
         repetition_us=FIR_rep,
-        ph1_cyc=IR_ph1_cyc,
-        ph2_cyc=IR_ph2_cyc,
         SW_kHz=config_dict["SW_kHz"],
```

Comment:
when I rearrange like this, its so the calls to these functions can be easily diff'd

Reply:
okay makes sense!